completions/puppet

```diff
@@ -168,7 +168,26 @@ _puppet()
                     return
             esac
             ;;
-        cert)
+        catalog)
+        case $prev in
```

Comment:
Indentation for the added catalog) block contents seems to be off, please follow style of existing blocks.